utils/envvars/parser.go

```diff
@@ -135,6 +142,33 @@ func parseStringDuration(value string) (time.Duration, error) {
 	return d, nil
 }
 
+// GetSAASHost returns SAAS host env variable value.
+// if it's valid, otherwise returns defaultSAASHost.
```

Comment:
The sentence should start from capital letter.

Reply:
updated